__tests__/reducers/index.test.js

```diff
@@ -314,26 +315,62 @@ describe('refreshResourceTemplate', () => {
     })
   })
 
-  it('missing reduxPath in payload should return the state', () => {
-    const missingPayload = refreshResourceTemplate({}, {
```

Comment:
AFAIKT there is not a way for the `refreshResourceTemplate` function to get a missing payload, and it would otherwise have to add extra lines of code to check for the existence of `Object.keys`